neuron_morphology/pipeline/_schemas.py

```diff
@@ -0,0 +1,89 @@
+from argschema.schemas import ArgSchema, DefaultSchema
+from argschema.fields import (
+    InputFile, OutputFile, String, Nested, Dict, List, Int, Field, Float)
+from marshmallow import ValidationError
+
+
+class PathResolution(DefaultSchema):
+    path = String(
+        description="line path", 
+        required=True
+    )
+
+    resolution = Float(
+        description="resolution",
+        required=True
+    )
+
+
+class PrimaryBoundaries(DefaultSchema):
+    Soma = Nested(
+        PathResolution,
+        description="soma drawing", 
+        required=True
+    )
+
+    White_Matter = Nested(
+        PathResolution,
+        description="white matter drawing",
+        required=False
+    )
+
+    Pia = Nested(
+        PathResolution,
+        description=("pia drawing"),
+        required=False
+    )
+
+
+class InputParameters(ArgSchema):
+    s3_bucket_uri = String(
```

Comment:
you mentioned earlier this is not actually a uri, but a short-form name. Is that so?

Reply:
should be a bucket_name. I will work with Sergey to make changes.

Reply:
should be s3_bucket_name